### ISSUE TITLE
  For some reason I cannot turn off gBr...

### ISSUE THREAD

**Issue by wormbasehelp** [WormBase/website#4592]
*Submitted from the feedback form on the WormBase website.*

<p>&nbsp;&nbsp;For some reason I cannot turn off gBrowse tracks, at least not using Safari - I go to "Select Tracks" and turn them off, but they are still there when I go back to the GBrowse view. May be a cookie issue? They are off when I use Chrome.</p>

**Reported by:** Abby********* (afde**************)
**Submitted from:** <a target="_blank" href="http://www.wormbase.org//tools/support">/tools/support</a>
**Browser:** Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_3) AppleWebKit/601.4.4 (KHTML, like Gecko) Version/9.0.3 Safari/601.4.4

**Comment by raymond91125** [comment]
Hi Abby,

Just to add that, in case you don't already know, conveniently on 
GBrowse, to the top left of each track, there is a row of little icons. 
If you tick on the one that looks like "X", it should turn off that 
specific track. To get back that track, you'll have to go to the "Select 
Tracks" page.

Cheers,

Raymond


On 02/18/2016 12:18 PM, Abby Dernburg wrote:
> Hi Chris -
>
> thanks for checking on this.
> I just went through the Select Tracks and systematically clicked “all 
> off” in every category and was finally able to get rid of them, so I 
> must have failed to do so earlier (I tried to scan the page for tracks 
> that were still on and couldn’t see anything). If you weren’t in the 
> process of switching to another browser I’d suggest making it easier 
> to find/turn off tracks that are on.
>
> Abby
>
> Abby Dernburg, Ph.D.
> HHMI Investigator
> Department of Molecular and Cell Biology
> University of California, Berkeley
> Berkeley, CA 94720
> USA
> +1-510-427-0199 (mobile)
>
>
>
>> On Feb 18, 2016, at 12:07 PM, Grove, Christian A. (Chris) 
>> <cgrove@caltech.edu <mailto:cgrove@caltech.edu>> wrote:
>>
>>
>> Hello Abby,
>>
>>  I cannot reproduce the error you describe on Safari. Can you let me 
>> know exactly which tracks are giving you the problem? If we can 
>> reproduce the error we will create a ticket for the GBrowse team to 
>> work on.
>>
>> Also, FYI, we are in the process of moving over to JBrowse as a 
>> browser; you may have better luck there, or not in which case please 
>> let us know.
>>
>> Thanks,
>>
>> Chris Grove
>> WormBase
>>
>>
>> ------------------------------------------------------------------------
>> *From:*no-reply@wormbase.org 
>> <mailto:no-reply@wormbase.org><no-reply@wormbase.org 
>> <mailto:no-reply@wormbase.org>>
>> *Sent:*Thursday, February 18, 2016 11:16 AM
>> *To:*help@wormbase.org <mailto:help@wormbase.org>
>> *Cc:*afdernburg@lbl.gov <mailto:afdernburg@lbl.gov>
>> *Subject:*[wormbase-help] Bug Report: For some reason I cannot turn 
>> off gBr... (Abby Dernburg)
>> Bug Report
>>   For some reason I cannot turn off gBrowse tracks, at least not 
>> using Safari - I go to "Select Tracks" and turn them off, but they 
>> are still there when I go back to the GBrowse view. May be a cookie 
>> issue? They are off when I use Chrome.
>> Submitted From:
>>   support/tools/support <http://www.wormbase.org//tools/support>
>> This issue has been assigned ticket number: #4592. Follow progress on 
>> ourissue tracker <https://github.com/WormBase/website/issues/4592>.
>> Reported by: Abby Dernburg (afdernburg@lbl.gov 
>> <mailto:afdernburg@lbl.gov>)
>> 18 Feb 16 19:19
>>
>> Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_3) AppleWebKit/601.4.4 
>> (KHTML, like Gecko) Version/9.0.3 Safari/601.4.4
>> This question was reported via the web interface atwormbase.org 
>> <http://wormbase.org/>. You can alsoemail us 
>> <mailto:help@wormbase.org>directly.
>> --
>> --
>> You received this message because you are subscribed to the 
>> "WormBase: help" group.
>> To post to this group, send email tohelp@wormbase.org 
>> <mailto:help@wormbase.org>
>> To unsubscribe from this group, send email to
>> help+unsubscribe@wormbase.org <mailto:help+unsubscribe@wormbase.org>
>> For more options, visit this group at
>> http://groups.google.com/a/wormbase.org/group/help?hl=en
>>
>> ---
>> You received this message because you are subscribed to the Google 
>> Groups "WormBase: help" group.
>> To unsubscribe from this group and stop receiving emails from it, 
>> send an email tohelp+unsubscribe@wormbase.org 
>> <mailto:help+unsubscribe@wormbase.org>.
>
> -- 
> -- 
> You received this message because you are subscribed to the "WormBase: 
> help" group.
> To post to this group, send email to help@wormbase.org
> To unsubscribe from this group, send email to
> help+unsubscribe@wormbase.org
> For more options, visit this group at
> http://groups.google.com/a/wormbase.org/group/help?hl=en
>
> ---
> You received this message because you are subscribed to the Google 
> Groups "WormBase: help" group.
> To unsubscribe from this group and stop receiving emails from it, send 
> an email to help+unsubscribe@wormbase.org 
> <mailto:help+unsubscribe@wormbase.org>.


-- 
__
Raymond Lee
raymond@caltech.edu
http://www.WormBase.org/
California Institute of Technology
Division of Biology and Biological Engineering
Mail Code 156-29
Pasadena, CA 91125
USA
tel: 626.395.2687

**Comment by raymond91125** [comment]
Hi Chris -

thanks for checking on this. 
I just went through the Select Tracks and systematically clicked “all off” in every category and was finally able to get rid of them, so I must have failed to do so earlier (I tried to scan the page for tracks that were still on and couldn’t see anything). If you weren’t in the process of switching to another browser I’d suggest making it easier to find/turn off tracks that are on.

Abby

Abby Dernburg, Ph.D.
HHMI Investigator
Department of Molecular and Cell Biology
University of California, Berkeley
Berkeley, CA 94720
USA 
+1-510-427-0199 (mobile)
